cabal-install/tests/IntegrationTests2.hs

```diff
@@ -53,6 +53,7 @@ import Distribution.Simple.Setup (toFlag, HaddockFlags(..), defaultHaddockFlags)
 import Distribution.Client.Setup (globalCommand)
 import Distribution.Simple.Compiler
 import Distribution.Simple.Command
+import qualified Distribution.Simple.Flag as Flg
```

Comment:
Have you used `Flg` on purpose? does it align with other qualified uses of the module in other placs of the codebase?
I would use Flag if the existing codebase dont use Flg

Reply:
There isn't another qualified import of flag. I just used that so it wouldn't be confused with the `Flag` Datatype. I've changed it now to "Flag" because "Flag.Flag" still makes sense.